tests/test_quixstreams/test_dataframe/test_windows/test_base.py

```diff
@@ -16,12 +18,9 @@ class TestGetWindowRanges:
         ],
     )
     def test_get_window_ranges_with_step(self, timestamp, duration, step, expected):
-        assert (
-            get_window_ranges(
-                timestamp_ms=timestamp, duration_ms=duration, step_ms=step
-            )
-            == expected
-        )
+        assert get_window_ranges(
+            timestamp_ms=timestamp, duration_ms=duration, step_ms=step
+        ) == deque(expected)
```

Comment:
Let's rather do `list(get_window_ranges(...))` here so we can freely change the return type as long as it's iterable without changing the test

Reply:
done